crypto/fipsmodule/service_indicator/service_indicator_test.cc

```diff
@@ -536,28 +536,40 @@ static const uint8_t kAESGCMCiphertext_256[64] = {
 };
 
 struct AEADTestVector {
+  const char* name;
   const EVP_AEAD *cipher;
   const uint8_t *key;
   const int key_length;
   const uint8_t *expected_ciphertext;
   const int cipher_text_length;
   const int expect_approved;
+  const bool test_repeat_nonce;
 } nAEADTestVectors[] = {
     // Internal IV usage of AES-GCM is approved.
-    { EVP_aead_aes_128_gcm_randnonce(), kAESKey, 16, nullptr, 0, AWSLC_APPROVED },
-    { EVP_aead_aes_256_gcm_randnonce(), kAESKey_256, 32, nullptr, 0, AWSLC_APPROVED },
+    { "AES-GCM 128-bit key internal iv test", EVP_aead_aes_128_gcm_randnonce(),
+        kAESKey, 16, nullptr, 0, AWSLC_APPROVED, false },
+    { "AES-GCM 256-bit key internal iv test", EVP_aead_aes_256_gcm_randnonce(),
+        kAESKey_256, 32, nullptr, 0, AWSLC_APPROVED, false },
     // External IV usage of AES-GCM is not approved unless used within a TLS
     // context.
-    { EVP_aead_aes_128_gcm(), kAESKey, 16, kAESGCMCiphertext_128, 64, AWSLC_NOT_APPROVED },
-    { EVP_aead_aes_192_gcm(), kAESKey_192, 24, kAESGCMCiphertext_192, 64, AWSLC_NOT_APPROVED },
-    { EVP_aead_aes_256_gcm(), kAESKey_256, 32, kAESGCMCiphertext_256, 64, AWSLC_NOT_APPROVED },
+    {  "Generic AES-GCM 128-bit key external iv test", EVP_aead_aes_128_gcm(),
+        kAESKey, 16, kAESGCMCiphertext_128, 64, AWSLC_NOT_APPROVED, false },
+    {  "Generic AES-GCM 128-bit key external iv test", EVP_aead_aes_192_gcm(),
+        kAESKey_192, 24, kAESGCMCiphertext_192, 64, AWSLC_NOT_APPROVED, false },
+    {  "Generic AES-GCM 128-bit key external iv test", EVP_aead_aes_256_gcm(),
```

Comment:
```suggestion
    {  "Generic AES-GCM 192-bit key external iv test", EVP_aead_aes_192_gcm(),
        kAESKey_192, 24, kAESGCMCiphertext_192, 64, AWSLC_NOT_APPROVED, false },
    {  "Generic AES-GCM 256-bit key external iv test", EVP_aead_aes_256_gcm(),
```

Reply:
My bad, got caught redhanded copying and pasting :)